index.bs

```diff
@@ -940,7 +977,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
         2.  Let |value| be a new {{PublicKeyCredential}} associated with |global| whose fields are:
 
             :   {{PublicKeyCredential/[[identifier]]}}
-            ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of the credential ID
+            ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of the [=credential ID=]
```

Comment:
if we use credential source to refer to the public key/private key, the credential ID should be a credential source ID.